pkg/receiver/smartagent/README.md

```diff
@@ -2,50 +2,49 @@
+1. To replace or modify metrics, use
 [Collector processors](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/README.md).
-1. Monitors with [dimension property and tag update
-functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
-allow an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.  These monitors include `ecs-metadata`, `heroku-metadata`, `kubernetes-cluster`, `openshift-cluster`, `postgresql`,
-and `sql`.
-If you do not specify any exporters via this field, the receiver will attempt to use the associated
-pipeline.  If the next element of the pipeline isn't compatible with the dimension update behavior, and if you configured
-a single SignalFx exporter for your deployment, the exporter will be selected.  If no dimension update behavior is desired,
-you can specify the empty array `[]` to disable.
-1. Monitors with [event-sending
-functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events) should also be made members of
-a `logs` pipeline that utilizes a [SignalFx
-exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
-that will make the event submission requests. It's recommended, and in the case of the Processlist monitor required,
-to use a [Resource Detection
-processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md)
-to ensure that host identity and other useful information is made available as event dimensions.
-Receiver entries that should be added to logs pipelines include `kubernetes-events`, `nagios`, `processlist`, and potentially any
-`telegraf/*` monitors like `telegraf/exec`.  The `signalfx` exporter is required for sending events to SignalFx, the `splunk_hec` exporter does not support sending events. An example of this is provided below.
+1. If you have a monitor that updates [dimension properties or tags](https://dev.splunk.com/observability/docs/datamodel/metrics_metadata)
+(e.g. `ecs-metadata`, `heroku-metadata`, `kubernetes-cluster`, `openshift-cluster`, `postgresql`, or `sql`), put the name of
+your SignalFx exporter in its `dimensionClients` field in the Collector's SignalFx receiver configuration block.
+If you do not specify any exporters in this array field, the receiver will attempt to use the Collector pipeline to which it's connected. If
+the next element of the pipeline isn't compatible with updating dimensions, and if you configured a single SignalFx exporter,
+the reciever will use that SignalFx exporter. If you don't require dimension updates, you can specify the empty array `[]` to disable it.
+1. If you have a monitor that sends [events](https://dev.splunk.com/observability/docs/datamodel/custom_events) (e.g. `kubernetes-events`,
```

Comment:
mind moving this up one in importance since this is required and the dimension client logic has automatic defaults?

Reply:
Sure, done.